njams-sdk-sample-client/src/main/java/com/faizsiegeln/test/SimpleClient.java

```diff
@@ -139,4 +140,43 @@ private static Settings getJmsProperties() {
         communicationProperties.put(JmsConstants.COMMANDS_DESTINATION, "njams4.dev.phillip.commands");
         return communicationProperties;
     }
+    
+        private static Settings getKafkaProperties() {
+	Settings communicationProperties = new Settings();
+	
+		//nJAMS properties
+        communicationProperties.put(CommunicationFactory.COMMUNICATION, "Kafka");
+        communicationProperties.put("idleCommandsResponseProducerTimeout", "300000"); //in ms
```

Comment:
Properties should follow the SDK naming scheme, i.e., using a unique prefix e.g., `njams.sdk.communication.kafka.destination`

Reply:
njams specific properties are modified now. Properties regarding Kafka are passed untouched to Kafka, so those keys need to follow Kafka guidelines.

Reply:
Kafka properties should also use the prefix as long as they are configured in the client's configuration.
Keys should be mapped to their plain Kafka equivalents when passed to the Kafka API.

Reply:
Changes Implemented